transformers/test_data/flip_file.go

```diff
@@ -0,0 +1,58 @@
+package test_data
+
+import (
+	"math/rand"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/event"
+	"github.com/makerdao/vulcanizedb/pkg/core"
+	"github.com/makerdao/vulcanizedb/pkg/fakes"
+)
+
+var rawFlipFileCatLog = types.Log{
+	Address: common.HexToAddress(FlipEthV110Address()),
+	Topics: []common.Hash{
+		common.HexToHash(constants.FlipFileCatSignature()),
+		common.HexToHash("0x000000000000000000000000be8e3e3618f7474f8cb1d074a26affef007e98fb"),
+		common.HexToHash("0x6361740000000000000000000000000000000000000000000000000000000000"),
+		common.HexToHash("0x0000000000000000000000000000000000000000000000000e4b4b8af6a70000"),
```

Comment:
If we're grabbing test data from another event, `cat_file_vow` might be a good choice (same signature, topics[3] is an address)